lib/route/route_obj.c

```diff
@@ -284,6 +289,9 @@ static void route_dump_details(struct nl_object *a, struct nl_dump_params *p)
 			r->rt_ttl_propagate ? "enabled" : "disabled");
 	}
 
+	if (r->ce_mask & ROUTE_ATTR_NHID && r->rt_nhid != 0)
+		nl_dump(p, "nhid %d ", r->rt_nhid);
```

Comment:
`%u`

that is appropriate for the type of `r->rt_nhid`, but it's also what iproute2 does.

Reply:
Fixed, and yes, %u sounds more appropriate.